src/schema/search.ts

```diff
@@ -228,6 +246,29 @@ export const resolvers: IResolvers<unknown, Context> = traceResolvers({
       ctx,
       info,
     ): Promise<ConnectionRelay<GQLPost> & { query: string }> => {
+      const gbClient = getUserGrowthBookInstace(ctx.userId);
```

Comment:
Our server side feature flag doesn't support the team attribute which will force us to use hardcoded user ids and bloat the boot endpoint. This is something we want to consider here. Can we perhaps send a search version from the client to `searchPosts` and `searchPostSuggestions` like we do with the feed? Will be much easier I believe